ext/opcache/ZendAccelerator.c

```diff
@@ -582,40 +562,46 @@ static void accel_use_shm_interned_strings(void)
 			info = (zend_property_info*)Z_PTR(q->val);
 
 			if (q->key) {
-				q->key = accel_new_interned_string(q->key);
+				q->key = new_interned_string(q->key);
 			}
 
 			if (info->name) {
-				info->name = accel_new_interned_string(info->name);
+				info->name = new_interned_string(info->name);
 			}
 		}
 
 		for (j = 0; j < ce->function_table.nNumUsed; j++) {
 			q = ce->function_table.arData + j;
 			if (Z_TYPE(q->val) == IS_UNDEF) continue;
 			if (q->key) {
-				q->key = accel_new_interned_string(q->key);
+				q->key = new_interned_string(q->key);
 			}
 			if (Z_FUNC(q->val)->common.function_name) {
-				Z_FUNC(q->val)->common.function_name = accel_new_interned_string(Z_FUNC(q->val)->common.function_name);
+				Z_FUNC(q->val)->common.function_name = new_interned_string(Z_FUNC(q->val)->common.function_name);
 			}
 		}
 
 		for (j = 0; j < ce->constants_table.nNumUsed; j++) {
 			q = ce->constants_table.arData + j;
 			if (Z_TYPE(q->val) == IS_UNDEF) continue;
 			if (q->key) {
-				q->key = accel_new_interned_string(q->key);
+				q->key = new_interned_string(q->key);
 			}
 		}
 	}
 
 	/* constant hash keys */
 	for (idx = 0; idx < EG(zend_constants)->nNumUsed; idx++) {
+		zend_constant *c;
+
 		p = EG(zend_constants)->arData + idx;
 		if (Z_TYPE(p->val) == IS_UNDEF) continue;
 		if (p->key) {
-			p->key = accel_new_interned_string(p->key);
+			p->key = new_interned_string(p->key);
+		}
+		c = (zend_constant*)Z_PTR(p->val);
+		if (c->name) {
```

Comment:
Probably not a usual case. Some tricks i recall were like giving a name with a \0-byte in the front, to save some thing unusual into the constant. Such code is for sure still somewhere in APCu and earlier in APC, probably in some other exts as wel.